web/src/components/atoms/ContentSection/index.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from "@emotion/styled";
```

Comment:
Please move ContentSection in to the InnerContents directory as it is a child component that is suppose to be used together with InnerContents

Reply:
@KaWaite , done.